src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Markup/MarkupExtension_ParserContext.xaml.cs

```diff
@@ -36,6 +36,26 @@ protected override object ProvideValue(IXamlServiceProvider serviceProvider)
 		{
 			return serviceProvider;
 		}
+#endif
+	}
+
+	public class HorOrientationExtension : MarkupExtension
+	{
+#if HAS_UNO || !WINDOWS_UWP // Winui,UnoUwp,UnoWinui: yes, Uwp: not-supported
+		protected override object ProvideValue(IXamlServiceProvider serviceProvider)
```

Comment:
could use this override that works everywhere:
```suggestion
		protected override object ProvideValue()
```